src/main/scala/xiangshan/frontend/ITTAGE.scala

```diff
@@ -148,7 +149,7 @@ class ITTageTable
   })
 
   val SRAM_SIZE=128
-  val nBanks = 2
+  val nBanks = 1
```

Comment:
Is nBanks not used anymore? Can we remove it?

Reply:
okk